eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/core/protocol/grpc/service/EventEmitter.java

```diff
@@ -18,8 +18,6 @@
 package org.apache.eventmesh.runtime.core.protocol.grpc.service;
 
 import io.grpc.stub.StreamObserver;
-
```

Comment:
The two imports need to be separated by a blank line as required by checkstyle.

Reply:
ok,3q